gobblin-yarn/src/main/java/org/apache/gobblin/yarn/YarnService.java

```diff
@@ -313,6 +313,21 @@ public void handleNewContainerRequest(NewContainerRequest newContainerRequest) {
         newContainerRequest.getResource());
   }
 
+  /**
+   *  getContainerInfoGivenHelixParticipant returns the containerInfo of the given helixParticipant if it exists else
+   *  return null
+   * @param helixParticipant
+   * @return ContainerInfo
+   */
```

Comment:
Please use this formatter for apache gobblin - https://gobblin.readthedocs.io/en/latest/developer-guide/CodingStyle/